deegree-services/deegree-webservices-handbook/src/main/asciidoc/appendix.adoc

```diff
@@ -0,0 +1,38 @@
+[[anchor-appendix]]
+== Appendix
+
+The following chapters of the documentation are aimed at users with specialized knowledge on the inner workings of deegree.
+
+NOTE: The functionality is currently in evolving status and may be change without future notice.

```

Comment:
Suggestion for a more explicit wording of this note:

As the internal workings of deegree are always evolving, the information in these chapters might become obsolete without any prior notice. There is no guarantee for anything described in here to be the same for future versions of deegree.

Reply:
+1